resources/datadog-agent/msi/localization-en-us.wxl.erb

```diff
@@ -63,4 +67,15 @@
   <String Id="WillInstall">[WixBundleName] will be installed on your device</String>
   <String Id="InstallLicenseLinkText">By installing you accept these &lt;a href="#"&gt;license terms&lt;/a&gt;</String>
 
+  <!-- these strings are for the main agent service install -->
+  <String Id="ServiceDisplayName">[ProductName]</String>
+  <String Id="ServiceDescription">Send metrics to [ShortCompanyName]</String>
```

Comment:
`CompanyNameShort` ?

Reply:
ShortCompanyName (now) property defined in source.wxs.erb.